README.md

```diff
@@ -189,15 +221,19 @@ export DOCKER_BUILDKIT=1
 tutor images build -a BUILDKIT_INLINE_CACHE=1 --docker-arg="--cache-from" --docker-arg="<docker-user>/<docker-repository>:<image-tag>" -a EDX_PLATFORM_REPOSITORY=<your-edx-repo> -a EDX_PLATFORM_VERSION=<branch> openedx
 ```
 
+### SUGGESTION: why is this needed? instead of using tutor images build ###
```

Comment:
It's based on the tutor documentation but I think I can refer to the official tutor documentation and get rid of this section